### PR TITLE
fix : remove empty spaces & unused code

### PR DESCRIPTION
**Pull request by bhaskarhc**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: src/app/table/table.component.ts [hunk 1]
```diff
@@ -224,44 +221,44 @@ export class TableComponent implements OnInit {
...
-          return "0 0";
-        }
-        else if (data[i].status == "failed") {
-          var count = 0;
-          for (var j = 0; j < data[i].jobs.length; j++) {
-            if(data[i].jobs[j].status == "success") {
-              count++;
-            }
+      if (data[i].status == "success") {
+        return "95 5";
```
Comment (inyee786) [inline src/app/table/table.component.ts]: what we are returning here??

File: src/app/workloads/workloads.component.ts [hunk 2]
```diff
@@ -443,7 +438,7 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
     this.captchaSuccess = true;
     this.captchaIsExpired = false;
     console.log(this.captchaSuccess);
-    (<HTMLInputElement> document.getElementById("litmusGo")).style.visibility = "visible";
+    (<HTMLInputElement>document.getElementById("litmusGo")).style.visibility = "visible";
     // (<HTMLInputElement> document.getElementById("captchaform")).disabled =true;
```
Comment (inyee786) [inline src/app/workloads/workloads.component.ts]: remove this

File: src/app/table/table.component.ts [hunk 3]
```diff
@@ -109,25 +106,25 @@ export class TableComponent implements OnInit {
         this.detailPannel(0, this.packetV11Data.dashboard, this.buildData.dashboard, 'packetv11');
       }
     });
-    
+
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: This line is not required

File: src/app/table/table.component.ts (hunk 1 above)
Comment (chandankumar4) [reply src/app/table/table.component.ts]: This is returing the value to creating the graph

File: src/app/table/table.component.ts [hunk 4]
```diff
@@ -371,417 +368,417 @@ export class TableComponent implements OnInit {
...
+    if (statusObj.successJobCount == statusObj.jobCount) {
       statusObj.status = "Success"
-    } else if (statusObj.skippedJobCount == statusObj.jobCount){
+    } else if (statusObj.skippedJobCount == statusObj.jobCount) {
       statusObj.status = "Skipped"
     } else if (statusObj.failedJobCount > 0) {
       statusObj.status = "Failed"
     }
-   
+
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: These two line is not required

File: src/app/table/table.component.ts [hunk 5]
```diff
@@ -371,417 +368,417 @@ export class TableComponent implements OnInit {
...
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Here also two line is not required

File: src/app/table/table.component.ts [hunk 6]
```diff
@@ -371,417 +368,417 @@ export class TableComponent implements OnInit {
...
       }
     }
-    for(var i = 0; i < data.length; i++) {
-      if(data[i].stage == "APP-CHAOS-TEST" && data[i].status == "success") {
+    for (var i = 0; i < data.length; i++) {
+      if (data[i].stage == "APP-CHAOS-TEST" && data[i].status == "success") {
         statusObj.successJobCount++;
       }
-      
+
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: This line also not required

File: src/app/table/table.component.ts [hunk 7]
```diff
@@ -371,417 +368,417 @@ export class TableComponent implements OnInit {
...
+    } else if (statusObj.skippedJobCount == statusObj.jobCount) {
       statusObj.status = "Skipped"
     } else if (statusObj.failedJobCount > 0) {
       statusObj.status = "Failed"
     }
-   
+
 
     return statusObj;
   }
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: remove this two line also

File: src/app/table/table.component.ts [hunk 8]
```diff
@@ -371,417 +368,417 @@ export class TableComponent implements OnInit {
...
       }
     }
-    for(var i = 0; i < data.length; i++) {
-      if(data[i].stage == "PROVIDER-INFRA-SETUP" && data[i].status == "success") {
+    for (var i = 0; i < data.length; i++) {
+      if (data[i].stage == "PROVIDER-INFRA-SETUP" && data[i].status == "success") {
         statusObj.successJobCount++;
       }
-      
+
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Remove this line also

File: src/app/table/table.component.ts [hunk 9]
```diff
@@ -371,417 +368,417 @@ export class TableComponent implements OnInit {
...
+    if (statusObj.successJobCount == statusObj.jobCount) {
       statusObj.status = "Success"
-    } else if (statusObj.skippedJobCount == statusObj.jobCount){
+    } else if (statusObj.skippedJobCount == statusObj.jobCount) {
       statusObj.status = "Skipped"
     } else if (statusObj.failedJobCount > 0) {
       statusObj.status = "Failed"
     }
-   
+
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Remove here two line

File: src/app/table/table.component.ts [hunk 10]
```diff
@@ -371,417 +368,417 @@ export class TableComponent implements OnInit {
...
       }
     }
-    for(var i = 0; i < data.length; i++) {
-      if(data[i].stage == "STATEFUL-APP-DEPLOY" && data[i].status == "success") {
+    for (var i = 0; i < data.length; i++) {
+      if (data[i].stage == "STATEFUL-APP-DEPLOY" && data[i].status == "success") {
         statusObj.successJobCount++;
       }
-      
+
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Remove this line

File: src/app/table/table.component.ts [hunk 11]
```diff
@@ -371,417 +368,417 @@ export class TableComponent implements OnInit {
...
+    if (statusObj.successJobCount == statusObj.jobCount) {
       statusObj.status = "Success"
-    } else if (statusObj.skippedJobCount == statusObj.jobCount){
+    } else if (statusObj.skippedJobCount == statusObj.jobCount) {
       statusObj.status = "Skipped"
     } else if (statusObj.failedJobCount > 0) {
       statusObj.status = "Failed"
     }
-   
+
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Remove the extra line from all the codes

File: src/app/workloads/workloads.component.ts [hunk 12]
```diff
@@ -403,20 +398,20 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
         if (this.litmusLog.completesstatus == true) {
           this.litmusStarted = false;
           this.litmusGoBtn = false;
-          
+
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.ts]: Remove this line

File: src/app/workloads/workloads.component.ts [hunk 13]
```diff
@@ -443,7 +438,6 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
     this.captchaSuccess = true;
     this.captchaIsExpired = false;
     console.log(this.captchaSuccess);
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.ts]: Why we are putting console here?

Comment (bhaskarhc) [reply src/app/workloads/workloads.component.ts]: for test purpose  , now i remove this !!

**Comment by chandankumar4** [comment]
@bhaskarhc Can you merge both commit into one commit

**Pull request merged**